src/network/networkd-address.c

```diff
@@ -1035,7 +1035,7 @@ void prefix_free(Prefix *prefix) {
 }
 
 int prefix_new(Prefix **ret) {
-        Prefix *prefix = NULL;
+        _cleanup_prefix_free_ Prefix *prefix = NULL;
```

Comment:
`prefix->radv_prefix` could be uninitialized when `prefix_free` is called, so it seems `prefix_free` should check that `prefix->radv_prefix` is not `NULL` in the same way it is done for `prefix->network`.

Reply:
Never mind. `sd_radv_prefix_unref` takes care of it. Thank you for fixing this.